ok, got a new one:

> Run sudo xcode-select --switch /Applications/Xcode_12.5.1.app
> xcode-select: error: invalid developer directory '/Applications/Xcode_12.5.1.app'
> Error: Process completed with exit code 1.

which I also see happening in `trunk` for a commit after the XCode update https://github.com/WordPress/gutenberg/runs/3940696898

--- a/phpunit/class-wp-theme-json-resolver-test.php
+++ b/phpunit/class-wp-theme-json-resolver-test.php
@@ -283,10 +283,24 @@ class WP_Theme_JSON_Resolver_Gutenberg_Test extends WP_UnitTestCase {
 							),
 						),
 					),
-					'custom'  => true,
+					'custom'  => false,
+					'link'    => true,
 				),
 				'blocks' => array(
-					'core/paragraph' => array(
+					'core/paragraph'  => array(
+						'color' => array(
+							'palette' => array(
+								'theme' => array(
+									array(
+										'slug'  => 'light',
+										'name'  => 'Light',
+										'color' => '#f5f7f9',
+									),
+								),
+							),
+						),
+					),
+					'core/post-title' => array(
 						'color' => array(
 							'palette' => array(
 								'theme' => array(

--- a/phpunit/data/themedir1/fse-child/theme.json
+++ b/phpunit/data/themedir1/fse-child/theme.json
@@ -19,10 +19,10 @@
 					"color": "#111827"
 				}
 			],
-			"custom": true
+			"link": true
 		},
 		"blocks": {
-			"core/paragraph": {
+			"core/post-title": {
 				"color": {
 					"palette": [
 						{

